website/source/docs/platform/k8s/helm/examples/kubernetes-auth.html.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "docs"
+page_title: "Using Kubernetes Auth Method"
+sidebar_current: "docs-platform-k8s-examples-kubernetes-auth"
+sidebar_title: "Using Kubernetes Auth Method"
+description: |-
+  Describes how to set up Kubernetes Auth method
+---
+
+# Bootstrapping Kubernetes Auth Method
+
+~> **Important Note:** This chart is not compatible with Helm 3. Please use Helm 2 with this chart.
+
+In this example, we will walk through how to set up the [Kubernetes Auth Method](/docs/auth/kubernetes.html).
+
+This assumes the following commands will be run inside a Vault pod running in Kubernetes.
+
+You will optionally need the following variables:
```

Comment:
These variables aren't needed, the command below is all they need in this situation.

Reply:
@pcman312 ^